CONTRIBUTING.md

```diff
@@ -2,6 +2,36 @@
 
 If you're reading this, you're awesome! Thank you for helping us make this project great and being a part of the Material-UI community. Here are a few guidelines that will help you along the way.
 
+## Git workflow
+
+1. Fork the Material-UI repository on GitHub
+
+2. Clone the fork to your local machine and add upstream remote
```

Comment:
Auto-increment, it will avoid future mistakes.
```suggestion
1. Clone the fork to your local machine and add upstream remote
```

Reply:
I don't think this is possible to use, because between numbered items there are non-numbered lines.